backend_new/test/integration/listing.e2e-spec.ts

```diff
@@ -216,4 +238,550 @@ describe('Listing Controller Tests', () => {
     expect(res.body.length).toEqual(1);
     expect(res.body[0].name).toEqual(listingA.name);
   });
+
+  it("should error when trying to delete listing that doesn't exist", async () => {
+    const id = randomUUID();
+    const res = await request(app.getHttpServer())
+      .delete(`/listings`)
+      .send({
+        id: id,
+      } as IdDTO)
+      .expect(404);
+    expect(res.body.message).toEqual(
+      `listingId ${id} was requested but not found`,
+    );
+  });
+
+  it('should delete listing', async () => {
+    const jurisdictionA = await prisma.jurisdictions.create({
+      data: jurisdictionFactory(),
+    });
+    const listingData = await listingFactory(jurisdictionA.id, prisma);
+    const listing = await prisma.listings.create({
+      data: listingData,
+    });
+
+    const res = await request(app.getHttpServer())
+      .delete(`/listings/`)
+      .send({
+        id: listing.id,
+      } as IdDTO)
+      .expect(200);
+
+    expect(res.body.success).toEqual(true);
```

Comment:
should there also be an expect to make sure the listing was actually deleted? Maybe a call to prisma to make sure the listing is no longer there?

Reply:
added!